js/web/lib/wasm/proxy-wrapper.ts

```diff
@@ -79,7 +79,7 @@ const onProxyWorkerMessage = (ev: MessageEvent<OrtWasmMessage>): void => {
   }
 };
 
-const scriptSrc = isProxy() ? (document?.currentScript as HTMLScriptElement)?.src : undefined;
+const scriptSrc = typeof document !== 'undefined' ? (document?.currentScript as HTMLScriptElement)?.src : undefined;
```

Comment:
It's possible that `isProxy()` read flags after ORT is loaded, when `ort.env.wasm.proxy` may change its value.

simply use `typeof document !== 'undefined'` is safe.